ui/app/styles/components/recovery.scss

```diff
@@ -0,0 +1,17 @@
+@use '../utils/size_variables';
+
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+.snapshot-card-container {
```

Comment:
I'm not sure we need a custom .scss file for this. Looking at the designs the padding for the card is `16px` all around and so the general css class `has-padding-m` ([here](https://github.com/hashicorp/vault/blob/main/ui/app/styles/helper-classes/spacing.scss#L53)) can be used directly on the card. 

We want to try to use general css class in favor of custom css unless the component needs a lot of custom styling. This way things like spacing are more consistent.